.travis.yml

```diff
@@ -1,17 +1,22 @@
+
 language: node_js
-dist: trusty
 sudo: required
-node_js: stable
+dist: trusty
+node_js: 8
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
```

Comment:
I presume this is just temporary until we can bump the dependency in `package.json`?

Reply:
yes - I'm not sure if I should include this at all - but it should fix the travis build/test for now (once the strongcheck issue got fixed upstream)